src/20_Components/amp-inputmask.html

```diff
@@ -0,0 +1,66 @@
+<!---{
+    "experiments": ["amp-inputmask"],
+    "preview": "default"
+}--->
+<!--
+  ## Introduction
+
+  Add input masks to inputs in your amp-forms. At this time, amp-inputmask
```

Comment:
add "`" to amp-inputmask and possibly everywhere yoy use it in the docs below

Reply:
ago